it/mail/chrome/messenger/safeMode.dtd

```diff
@@ -43,26 +43,26 @@
 <!ENTITY safeModeOptionsWindow.width       "width: 49em;">
 <!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
+<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting
 with one or more of these trouble shooting options selected:">
 
 
-<!ENTITY disableAddons.label              "Permamently Disable all Add-ons">
```

Comment:
Please check the English source text and replace "Permamently" with "Permanently".